docs/pages/versions/unversioned/guides/adhoc-builds.md

```diff
@@ -157,6 +160,55 @@ An invalid profile can be caused by revoking the distribution certificate or dis
 
 ![Invalid Profile](/static/images/adhoc-builds-invalid-profile.png)
 
+# Frequently Asked Questions
+
+## Distribution certificate CLI options
+
+`expo-cli` will give you options to produce a distribution certificate, and this section describes these options in more detail.
+
+### Creating a new certificate (recommended)
+
+This will create a new iOS distribution certificate with a password. This option won't be available if you've already reached Apple's limits of 2 active certificates in your development account.
+
+### Revoking existing certificates
+
+Revoking an existing distribution certificate associated with an App is safe if:
+
+- Your App is distributed through the App Store and you are **NOT** on an Apple Enterprise account.
+
+Revoking an existing distribution certificate associated with an App is **NOT** safe if:
+
+- Your App is distributed through the App Store and you are on an Apple Enterprise account.
+- Your App is distributed Ad hoc (distributed outside of the App Store for testing purposes).
+
+An App Store App gets re-signed with an Apple certificate when it goes on the store with non Enterprise accounts. Revoking the certificate therefore won't affect it. Enterprise Apps and Apps distributed Ad hoc use the original certificate, which means revoking it will cause the App to stop functioning on all devices it is installed on.
```

Comment:
```suggestion
An App Store App gets re-signed with an Apple certificate when it goes on the store with non-Enterprise accounts. Revoking the certificate therefore won't affect it. Enterprise Apps and Apps distributed ad hoc use the original certificate, which means revoking it will cause the App to stop functioning on all devices it is installed on.
```